akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/PersistentBehaviors.scala

```diff
@@ -66,14 +66,14 @@ object PersistentBehaviors {
      *
      * @see [[Effect]] for possible effects of a command.
      */
-    def command[Command, Event, State](commandHandler: Command ⇒ Effect[Event, State]): (ActorContext[Command], State, Command) ⇒ Effect[Event, State] =
```

Comment:
also in docs above: `(ActorContext[Command], State, Command) ⇒ Effect[Event, State]`

Reply:
and also some in reference docs, search for `ActorContext`

Reply:
perhaps the reference docs should also contain a new section about `ActorContext` including an example of how to obtain it via setup, so that it's clear that a persistent actor has the full capabilities of an ordinary actor if needed